### ISSUE TITLE
Update the Gnosis Safe URL 

### ISSUE THREAD

**Issue by blink-km** [PrimeDAO/prime-launch-dapp#706]
![IMAGE 2565-01-19 06:49:10.jpg](https://images.zenhubusercontent.com/6194d77c0f22bb6aa034c73b/b4dc1b58-ec58-4a97-9458-61d598860991)

At the pop-up before a user can submit an LBP launch, we have a link that is supposed to lead the user to Mainnet Gnosis Safe but it is currently linking the user to Rinkeby Gnosis Safe

**Comment by blink-km** [comment]
This is the updated URL:
https://gnosis-safe.io/app/eth:0x52F50f557704938Df066EC4Db7426D66538E7796/

**Comment by dkent600** [comment]
The FE is supply the correct URL as far as I can tell:

https://gnosis-safe.io/app/#/safes/0x52F50f557704938Df066EC4Db7426D66538E7796/transactions

"0x52F50f557704938Df066EC4Db7426D66538E7796" is the SAFE contract address given us by the contracs-v2 ABI json file for mainnet.

@Zitzak @codebuster22

**Comment by Zitzak** [comment]
Indeed, this is the safe on mainnet, address supplied by the ABIs. You can see the launch of the D2D token in its transactions. So this should just be the right URL

**Comment by codebuster22** [comment]
If you see, this url (https://gnosis-safe.io/app/#/safes/0x52F50f557704938Df066EC4Db7426D66538E7796/transactions) resolves to a different URL (https://gnosis-safe.io/app/eth:0xc54Ad3e73BDE767738335809A5f20F9BB80Be8D3/balances).

Now they use same domain for different network.

`https://gnosis-safe.io/app/{GnosisSpecificNetworkName}:{safeAddress}/`

Where,
`GnosisSpecificNetworkName` - A short name given to network by Gnosis. Eg. "eth" - for Ethereum mainnet, "rin" - Rinkeby Testnet


Now they use same domain for different network.

**Comment by dkent600** [comment]
> Where, `GnosisSpecificNetworkName` - A short name given to network by Gnosis. Eg. "eth" - for Ethereum mainnet, "rin" - Rinkeby Testnet

@codebuster22 Can you please point to documentation of this, where we can see the prefix values for each network?
